docs/DOC207.md

```diff
@@ -21,12 +21,34 @@ The documentation contains a `<see langword="..."/>` element with an unrecognize
 
 ## Rule description
 
-*TODO*
+A violation of this rule occurs when documentation contains a `see langword` element but the reference is not recognized
+as a language keyword.
+
+```csharp
+/// <summary>
+/// This variable is <see langword="signed">.
+/// </summary>
+public int signed;
+```
 
 ## How to fix violations
 
-*TODO*
+To fix a violation of this rule, replace the `see langword` syntax with the equivalent inline code.
+
+```csharp
+/// <summary>
+/// This variable is <c>signed</c>.
+/// </summary>
+public int signed;
+```
 
 ## How to suppress violations
 
-*TODO*
+```csharp
+#pragma warning disable DOC207 // Use see langword correctly
```

Comment:
💭 I would expect quotes around `see langword`. Might be good to check the messages and see what form they are currently in.

```suggestion
#pragma warning disable DOC207 // Use 'see langword' correctly
```